repository-data/application/src/main/resources/hcm-config/namespaces/hee/details.yaml

```diff
@@ -77,6 +84,20 @@ definitions:
             field: content
             plugin.class: org.hippoecm.frontend.editor.plugins.field.NodeFieldPlugin
             wicket.id: ${cluster.id}.field
+            hint: ''
+            /cluster.options:
+              jcr:primaryType: frontend:pluginconfig
+              ckeditor.config.appended.json: '{ removeButtons: "PickImage,Table",
```

Comment:
I think `ckeditor.config.appended.json` config isn't needed here as it would be inherited from `/hippo:namespaces/hippostd/html/editor:templates/_default_/@ckeditor.config.appended.json`. But, you may need to add `ckeditor.config.overlayed.json: '{ forcePasteAsPlainText: true }'` to enforce existing `forcePasteAsPlainText` functionality.